### DISCUSSION TITLE
zh/faq/error

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#229] [Announcements]
# zh/faq/error

这个是什么？ 这个是收集一些AList使用期间的一些错误码，然后如何解决（网络问题或者网盘更改API以及其它一些问题可能会无法当时解决需要人为去干预） ：阿里云盘open出现 TooManyRequests 、Too Many Requests ：点我查看详细说明 (../guide/drivers/aliyundrive_open.md) ：阿里云盘...

https://alist.nn.ci/zh/faq/error.html

**Comment by imtour** [comment]
failed get objs: failed to list objs: AccessTokenExpired:JWT is expired at `2023-05-23T05:29:05Z`，怎么处理

**Reply by anwen-anyi** [reply]
更新AList版本

**Comment by TG-GOD** [comment]
计入网页的时候出现这个：获取设置失败： 请稍候，正在加载储存

**Reply by wyf9** [reply]
可能是刚启动服务需要等存储加载完成
也可能是有存储出了问题

详见官方文档：使用指南 - 安装 - 一键脚本 / 手动安装 - `一直在加载怎么办?`

https://alist.nn.ci/zh/guide/install/manual.html#%E4%B8%80%E7%9B%B4%E5%9C%A8%E5%8A%A0%E8%BD%BD%E6%80%8E%E4%B9%88%E5%8A%9E

**Comment by megumi335400** [comment]
Post "https://api.nn.ci/alist/ali_open/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)  ，阿里云盘突然挂了，不知道怎么回事

**Reply by megumi335400** [reply]
重新扫码登录，换`token`也没用

**Comment by missjieq** [comment]
Post "https://api.nn.ci/alist/ali_open/token": dial tcp 45.66.130.152:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.拜托大神看看这个怎么解决

**Comment by ydz2021** [comment]
使用raidrive挂载时，raidrive错误代码401，浏览器打开同链接显示Method Not Allowed

**Comment by tb2020y** [comment]
MAC上无法挂载到本地，已使用桌面版程序。

[GIN] 2023/07/09 - 20:59:54 | 200 |    2.940548ms |       127.0.0.1 | OPTIONS  "/dav"
[GIN] 2023/07/09 - 20:59:57 | 200 |      35.931µs |       127.0.0.1 | OPTIONS  "/dav"
[GIN] 2023/07/09 - 20:59:57 | 200 |      32.299µs |       127.0.0.1 | OPTIONS  "/dav/"
[GIN] 2023/07/09 - 20:59:57 | 401 |       1.807µs |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/07/09 - 20:59:57 | 207 |    2.242805ms |       127.0.0.1 | PROPFIND  "/dav/"

**Comment by ffeqech** [comment]
failed to start: listen tcp 0.0.0.0:5244: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

**Comment by lubenwei1114** [comment]
System error: SyntaxError: Invalid regular expression: /?/gm: Nothing to repeat这个是什么问题？

**Reply by anwen-anyi** [reply]
> System error: SyntaxError: Invalid regular expression: /?/gm: Nothing to repeat这个是什么问题？
> 

你可以拿这个错误在常见问题的为什么搜索一下

**Comment by PyerryQ** [comment]
invalid character '<' looking for beginning of value
failed init storage ,阿里云盘open错误

**Reply by anwen-anyi** [reply]
> invalid character '<' looking for beginning of value
> failed init storage ,阿里云盘open错误

https://github.com/alist-org/alist/issues/4840
暂时的办法你重新保存一下 或者右下角刷新一下试试看

**Reply by PyerryQ** [reply]
好的，已解决

**Comment by jotony** [comment]
我尝试在nginx反代了127.0.0.1:5244后，上传文件提示882#0: *35 sendfile() failed (32: Broken pipe) while sending request to upstream, 请问应该如何解决呢

**Reply by PuqiAR** [reply]
nginx反向代理设置最大文件大小，增加一些（好像叫max buf size)

**Comment by guihu5277** [comment]
Trigger security policy : Please try again later   这是什么问题

**Reply by anwen-anyi** [reply]
什么地方遇到的 什么场景 描述清楚。

**Comment by DukeAnn** [comment]
升级完成后打开网页 提示这个：获取设置失败： 请稍候，正在加载储存
等了半天也不行

**Reply by DukeAnn** [reply]
报错： 2023/08/27 14:11:37.645801 ERROR RESTY Post "https://api.nn.ci/alist/ali_open/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 3

**Reply by wyf9** [reply]
和上面一个人一样，可以看官方文档的：使用指南 - 安装 - 一键脚本 / 手动安装 - 一直在加载怎么办?

https://alist.nn.ci/zh/guide/install/manual.html#%E4%B8%80%E7%9B%B4%E5%9C%A8%E5%8A%A0%E8%BD%BD%E6%80%8E%E4%B9%88%E5%8A%9E

里面提到可以用命令禁用有问题的存储

**Comment by jigu168** [comment]
failed to connect database:disk I/O error: invalid argument   openwrt 里面出现的，剩余存储4M 这个对数据存储空间有最小要求么？

**Comment by DjangoYu66** [comment]
请问alist挂载根目录ID跟换后一直不起作用，显示的文件夹还是之前的，要怎么刷新呀

**Reply by anwen-anyi** [reply]
> 请问alist挂载根目录ID跟换后一直不起作用，显示的文件夹还是之前的，要怎么刷新呀

右下角刷新

**Comment by Hxfrzc** [comment]
Cannot read properties of undefined (reading 'status')
上传文件遇到这个怎么处理

**Comment by solcorvuss** [comment]
挂载迅雷云盘出现这个： Failed load storage: failed init storage: ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: meta.username expect 199XXXXX19, but got map[phone_number:+86 199XXXXX19 result:accept], please check captcha init params    
求大佬解答

**Reply by Johnson1618** [reply]
我今天也遇到这个情况，明明验证码是对的，但是就是不行

**Reply by uestcwangci** [reply]
我也遇到了，我的alist没有更新，似乎是迅雷的问题

**Comment by solcorvuss** [comment]
估计某些程序更新了导致旧版的不可用

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Johnson1618 ***@***.***>
发送时间: Thursday, September 14, 2023 5:28:41 PM
收件人: alist-org/docs ***@***.***>
抄送: gcorvuss ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/faq/error (Discussion #229)


我今天也遇到这个情况，明明验证码是对的，但是就是不行

―

**Comment by nkdns** [comment]
打包下载时候提示：
当前状态: 获取文件夹结构失败: failed get objs: failed to list objs: TooManyRequests:Throttled by PLUGIN Flow Control
现在是多文件结构可以下载、多文件夹结构就会报错

**Comment by GitHub-wp** [comment]
Alist挂载夸克网盘，然后用Aria-ng下载，大概几分钟左右就停止下载，登录VPS查看状态，然后就出现如下的代码；
周而复始的出现这个问题，
alist.service - Alist service
     Loaded: loaded (/etc/systemd/system/alist.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2023-09-28 02:05:40 UTC; 1min 10s ago
    Process: 194481 ExecStart=/opt/alist/alist server (code=exited, status=2)
   Main PID: 194481 (code=exited, status=2)

**Comment by huaxing211** [comment]
挂载天翼云盘：
https://open.e.189.cn/api/logbox/oauth2/unifyAccountLogin.do?appId=cloud&version=v1.1&clientType=1&format=redirect&paras=XXXXXX&sign=XXXXXXX": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
这个是什么问题？

**Comment by ahhszhcsssCN** [comment]
ailed to start http: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

浏览器端可以挂载，windows端不行，求教怎么办

**Reply by wyf9** [reply]
翻译：`启动 http 时出错： listen tcp 0.0.0.0：5244： bind： 通常只允许使用每个套接字地址（协议/网络地址/端口）一次。`

可能是有程序占用了 `5244` 端口导致AList无法使用

建议搜索: https://www.bing.com/search?q=ailed%20to%20start%20http%3A%20listen%20tcp%200.0.0.0%3A5244%3A%20bind%3A%20Only%20one%20usage%20of%20each%20socket%20address%20(protocol%2Fnetwork%20address%2Fport)%20is%20normally%20permitted.

**Comment by yangguangdijia** [comment]
我把alist的整个目录删掉了，重新安装PS D:\leyun> .\alist.exe server
INFO[2023-10-31 23:24:56] reading config file: data\config.json
INFO[2023-10-31 23:24:56] config file not exists, creating default config file
FATA[2023-10-31 23:24:56] failed to create config file: open data\config.json: Access is denied.出现这个错误，怎么解决

**Comment by TuoLingTeam** [comment]
System error: TypeError: n.replaceAll is not a function

**Comment by TuoLingTeam** [comment]
后台头部添加了：<script src="https://polyfill.io/v3/polyfill.min.js?features=String.prototype.replaceAll"></script>

**Comment by TuoLingTeam** [comment]
后台头部添加了：<script src="https://polyfill.io/v3/polyfill.min.js?features=String.prototype.replaceAll"></script>

还是有这个错误提示：System error: TypeError: n.replaceAll is not a function

**Comment by langvs** [comment]
请问，我使用的是docker支持aria2的版本，在哪里修改上传大小限制，我修改了宝塔的nginx的限制大小和反代的限制大小，可是上传还是提示413~难道要修改容器里面的东西吗？？

**Reply by langvs** [reply]
我修改了client_max_body_size 20000m;，反代配置文件里也设置了client_max_body_size 20000m;，还是提示
Request failed with status code 413

**Comment by zfwc** [comment]
wind系统 映射的时候提示这个 怎么弄
WebDAV (127.0.0.1)(Z:）
http://admin@127.0.0.1:5244路径:/dav原因:由于目标计算机积极拒绝，无法连接。 127.0.0.1:5244
请检查地址是否正确

**Reply by cy248740603** [reply]
新建一个用户别用admin试试

**Comment by Autolenuants** [comment]
挂载115一段时间后提示：很抱歉，由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。

**Comment by Seventeenslove** [comment]
[del_user_cache_online] failed: Post "http://localhost:5244/api/admin/user/del_cache?username=admin": dial tcp [::1]:5244: connect: connection refused
如何解决，换了一台linux也是这样

**Comment by blackedxyz** [comment]
https://i.ibb.co/zhBd8mT/2023-12-03-205258.jpg
帮忙看看这个报错是怎么回事？？
用的这个docker代码：docker run -d --restart=always -v /share/CACHEDEV1_DATA/Docker/Alist:/opt/alist/data -p 52441:52441 --name="alist" xhofe/alist:latest

**Reply by anwen-anyi** [reply]
没问题啊，就是这样 已经启动了
https://github.com/alist-org/alist/issues/5618

**Comment by Ufjcij** [comment]
failed get storage: storage not found; rawPath: /auth/sso_callback  我使用github登录alist出现这个是咋回事

**Comment by cy248740603** [comment]
阿里云盘复制文件到本地稍大的文件就会大量出现 response error: The network name was deleted 导致复制成功率很低

**Comment by kizhou** [comment]
failed get objs: failed to list objs: Trigger security policy : Please try again later

**Comment by Maxhit110** [comment]
登录成功后出现Token is expired ，没有使用CDN加速，请问如何处理？

**Comment by FrankSyf** [comment]
Failed find user: record not found
怎么解决

**Reply by FrankSyf** [reply]
回复一下吧，很急，要用AList

**Comment by kizhou** [comment]
Hi 
解决方法也挺简单的，更新到最新版本即可

From: FrankSyf
Date: 2024-01-04 20:19
To: alist-org/docs
CC: kizhou; Comment
Subject: Re: [alist-org/docs] zh/faq/error (Discussion #229)
回复一下吧，很急，要用AList

**Comment by xzyl4303** [comment]
出现database is locked怎么办

**Reply by kizhou** [reply]
请贴出图片，以方便解决问题。

**Comment by GJLR88** [comment]
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
下载 alist-linux-musl-amd64.tar.gz 失败！

安装的时候出现这个怎么解决

**Comment by Miyang0164** [comment]
2024/02/06 19:00:25 http: TLS handshake error from 192.168.50.1:35314: remote error: tls: unknown certificate
2024/02/06 19:00:25 http: TLS handshake error from 192.168.50.1:35320: remote error: tls: unknown certificate
2024/02/06 19:00:26 http: TLS handshake error from 192.168.50.1:35334: remote error: tls: unknown certificate
2024/02/06 19:00:28 http: TLS handshake error from 192.168.50.1:35342: remote error: tls: unknown certificate

这是手机 用edge浏览器访问 点本地视频 就会弹这个 并且无法观看 但是换成夸克浏览器访问 就能正常观看 
有什么办法解决吗 除了换浏览器0.0

**Comment by binwluo** [comment]
123云盘，failed link: failed get link: 流量超限

**Reply by wyf9** [reply]
https://alist.nn.ci/zh/guide/drivers/123.html

123云盘的挂载文档里说了：免费用户10G流量下载

应该是超出10g下载了所以提示超限

**Comment by linger6580** [comment]
failed get objs: failed to list objs: Get "https://api-drive.mypikpak.com/drive/v1/files?filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=&thumbnail_size=SIZE_LARGE&with_audit=true": read tcp ***********1:14436->***********7:443: wsarecv: An existing connection was forcibly closed by the remote host.
求教怎么解决啊！

**Comment by qq7171** [comment]
为啥打开alist是空白? 之前用的好好的,最近几天突然出现空白,重启docker也没有用,无论是内外网访问均是空白

**Comment by rubbon6** [comment]
Failed get storage driver: no mount path for an storage is,怎么处理，添加了个储存，状态什么都没有，也不能禁用，不能删除，点了就弹出来这个

**Comment by kizhou** [comment]
是不是路径错误了？如果还不行，私聊我QQ



---Original---
From: ***@***.***&gt;
Date: Mon, Mar 4, 2024 22:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] zh/faq/error (Discussion #229)




 
Failed get storage driver: no mount path for an storage is,怎么处理，添加了个储存，状态什么都没有，也不能禁用，不能删除，点了就弹出来这个

**Comment by jysniper** [comment]
挂载其他人alist，保存时报错Failed init storage: Get "/alist.xiaoya.pro/api/me": unsupported protocol scheme ""

**Reply by wyf9** [reply]
是不是没加 `http://` / `https://` 前缀？

错误提示机翻：`初始化存储失败：获取 “/alist.xiaoya.pro/api/me”：不支持的协议方案 “”`

**Comment by wangyunzi** [comment]
阿里云盘open出现这个报错failed get objs: failed to list objs: NotFound.ParentFileId:The resource parent_file_id cannot be found. parent_file_id:651e83171b807bdf4b45470483151b4abe460122 not found

**Comment by tianyouwuai** [comment]
百度网盘资源下载时跳出下面的代码，怎么办？
{
    "code": 500,
    "message": "http request [https://d.pcs.baidu.com/file/019108027v4da4eae679cfaebfaa3600?fid=3191170421-250528-238947460198729&rt=pr&sign=FDtAERK-DCb740ccc5511e5e8fedcff06b081203-zRsY6Ty7CTOTHv%2ffuxAIUjMpedI%3d&expires=8h&chkbd=0&chkv=0&dp-logid=9145400489965168165&dp-callid=0&dstime=1712337869&r=218471060&vuk=3191170421&origin=dlna] failure,status: 403 response:\u001f�\b\u0000\u0000\u0000\u0000\u0000\u0000\u0003-��\u000e� \f\u0000�_iz�@\u0001g�g\bJ�duf\u0005O�_����BV=4-Ga�����S{[1�V;̒�7��Uj�`��U���P��Ѡ����S-\u0018'\nC��ED�\u0005�d��\u0007ON\u0005Bi\u0000\u0000\u0000",
    "data": null
}

**Comment by 3JXC** [comment]
挂载夸克网盘，通过主页访问时弹出错误提示：“failed get objs: failed to list objs: query fail [61008]”，所有设置及cookie再三检查没有错误啊

**Comment by willows888** [comment]
start mounting...
2024/04/20 19:40:54 NOTICE: webdav root '': --vfs-cache-mode writes or full is recommended for this remote as it can't stream
2024/04/20 19:40:54 Fatal error: failed to mount FUSE fs: mount stopped before calling Init: mount failed: cgofuse: cannot find winfsp
mount finished with code 1 and signal null（出现在挂载驱动时）

**Comment by wustdjf** [comment]
request failed,code: 400, message: password is incorrect            挂载小雅出现错误

**Comment by floodstream** [comment]
FATA[2024-05-27 19:15:42] load config error: readObjectStart: expect { or n, but found , error found in #0 byte of ...||..., bigger context ...||...
[AD] Command finished with code 1 and signal null

**Comment by sunbingrun** [comment]
`failed get objs: storage not init: res_code: UserInvalidOpenToken ,res_msg: error in login get unifyAccountInfo is null`
天翼个人云，报这个错误求救

**Comment by pan1394** [comment]
使用mysql作为数据库， 获取密码时报错：
[root@OpenWrt:01:03 AM /etc/alist] # alist admin random
INFO[2024-06-27 17:03:21] reading config file: data/config.json        
INFO[2024-06-27 17:03:21] load config from env with prefix: ALIST_     
INFO[2024-06-27 17:03:21] init logrus...                               
INFO[2024-06-27 17:03:22] admin user has been updated:                 
INFO[2024-06-27 17:03:22] username: admin                              
INFO[2024-06-27 17:03:22] password: r9UjMFRY                           
ERRO[2024-06-27 17:03:22] [del_user_cache_online] error: that's not even a token

**Comment by RhysChina** [comment]
我挂载路径填的“/”根目录，也只能填一个嘛，不然会报Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Comment by luxavi** [comment]
阿里云挂在成功后下载失败权限不足

**Comment by AYOMITA** [comment]
您好，在我创建新文件时提示我“Upload not supported”，在我创建文件夹时提示我“Not implement”，环境是使用内网IP访问局域网服务器，登录为管理员帐号，服务器Alist.exe已为所有用户开启管理员运行，”本地存储”的权限是777，存储结构是使用“别名“从“本地存储”链接到“用户文件夹(也是本地存储)”，我从公网域名访问后，也是一样的结果。我想问一下可能是什么原因导致无法上传？

**Comment by bobmarley28** [comment]
您好，在115网盘更新之后，挂载就显示失败了，添加页面显示【failed to login by qrcode: {"state":0,"error":"登录失败，系统已下架","errno":0,"message":"登录失败，系统已下架","code":0}: unexpected error】，在文件页面显示【failed get objs: failed to list objs: {"state":false,"error":"登录超时，请重新登录。","errNo":990001,"request":"\/files?aid=1&asc=1&cid=0&fc_mix=0&format=json&limit=56&natsort=0&o=user_ptime&offset=0&record_open_time=1&show_dir=1&snap=0"}: user not login】，请教一下是什么原因？

**Reply by KBKUN024** [reply]
挂载的时候，本来是linux，改成网页版就能挂载了

**Comment by laoma2053** [comment]
点击下载，弹出如下错误代码：{"error_code":31329,"error_msg":"hit black userlist , hit illeage dlna","request_id":203206552361456464}

**Reply by laoma2053** [reply]
挂载的是百度网盘，系统版本升级到最新3.36

**Comment by FopFat** [comment]
今天打开网页，突然出现这个提示：System error: Error: [Hope UI]: useColorMode must be used within a HopeProvider
进不了页面设置。
求解。

**Comment by HereNoAg300L** [comment]
上传的时候遇到的错误代码
request failed, status: 403 Forbidden
因为用的是1panel搭建的，没研究好，希望大佬帮忙看一下，辛苦

**Reply by KBKUN024** [reply]
同样的问题，也是403，上传30GB的视频的时候碰到的，你解决了吗？

**Comment by Dryangxiansheng** [comment]
Fatal error: failed to mount FUSE fs: mount stopped before calling Init: mount failed: cgofuse: cannot find winfsp
mount finished with code 1 and signal null，挂载不上去呀

**Reply by ylj5096** [reply]
需要先安装winfsp，安装地址：https://mirror.ghproxy.com/https://github.com/winfsp/winfsp/releases/download/v2.0/winfsp-2.0.23075.msi

**Comment by KBKUN024** [comment]
我挂载的是115网盘，有vip，所以最大单个文件可以115GB，但是上传文件的时候碰到的一个问题，上传的是一个接近30GB的视频，之前上传十几GB的视频是可以上传成功的，然后这个视频突然不行，报错：
错误码：403，message:The security token you provided is invalid.请问要怎么解决？是我上传的文件太大了吗？

**Comment by jonehoo** [comment]
用对象存储挂载七牛云的时候报这个错误，failed get objs: failed to list objs: RequestError: send request failed caused by: Get "https://alist28718.s3.cn-north-1.qiniucs.com/?delimiter=%2F&marker=&prefix=": tls: failed to verify certificate: x509: failed to load system roots and no roots provided; open /etc/ssl/certs/ca-certificates.crt: input/output error

**Comment by LWDJD** [comment]
read F:\alist\alist-windows-amd64_3\data\temp\file-194372540: file already closed; CreateTempFile failed
批量上传的时候会出现一堆这样的报错，我随便上传了几个文件夹，然后产生的足足8页这样的报错，报错的文件也没上传上去，我用的alist自带的加密功能。

**Comment by xiaosa70** [comment]
从123盘复制到onedrive盘的时候一部分文件提示这个
{"error":{"code":"invalidRequest","message":"Invalid request"}}
想请教一下大佬怎么解决

**Comment by gonglh** [comment]
修改admin密码：.\alist admin set xxx，提示密码已更新，但最后一行打印了“[del_user_cache_online] error”错误，新密码无法登录。
完整输出：
PS D:\softs\alist-windows-amd64> .\alist admin set xxx
INFO[2024-10-12 11:50:57] reading config file: data\config.json
INFO[2024-10-12 11:50:57] load config from env with prefix: ALIST_
INFO[2024-10-12 11:50:57] init logrus...
INFO[2024-10-12 11:50:57] admin user has been updated:
INFO[2024-10-12 11:50:57] username: user
INFO[2024-10-12 11:50:57] password: xxx
ERRO[2024-10-12 11:50:57] [del_user_cache_online] error: token is invalidated
PS D:\softs\alist-windows-amd64>
